bugbug/model.py

```diff
@@ -386,11 +386,17 @@ def classify(
                 importance_cutoff, shap_values
             )
 
-            # TODO: pred class index
+            # Workaround: handle multi class case for force_plot to work correctly
+            if not len(classes[0]) == 2:
+                pred_class_index = classes.argmax(axis=-1)[0]
+                explainer.expected_value = explainer.expected_value[0]
+            else:
+                pred_class_index = 0
```

Comment:
@marco-c for binary case , we have values stored in  a single class i.e., ["class 0"], so we have to make the index 0, and not the one with maximum probas.